tensorflow/compiler/mlir/lite/tests/optimize.mlir

```diff
@@ -642,6 +642,34 @@ func @NotReorderReshapeAddIfHighDim(%arg0: tensor<1x1x1x1x30x96xf32>) -> tensor<
   // CHECK: return %[[rs2]]
 }
 
+// CHECK-LABEL: @ReorderReshapeAdd2DConst
```

Comment:
Could you add test cases for reordering and fusing?

Reply:
Updated. I follow the test cases of `FuseFullyConnectedReshapeAddConst*`. Let me know if that's applicable. Thank you!